src/zui/ZUIHeader/index.tsx

```diff
@@ -0,0 +1,290 @@
+                />
+              ) : (
+                title
+              )}
+            </Typography>
+          </Box>
+        </Box>
+        {showActionButton && (
+          <>
+            <ZUIButtonGroup
+              buttons={actionButtons}
+              variant={actionButtonVariant}
+            />
+            {!!actionButtonPopoverContent && (
+              <Popover
+                anchorEl={actionButtonPopoverAnchorEl}
+                anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
+                onClose={() => setactionButtonPopoverAnchorEl(null)}
+                open={!!actionButtonPopoverAnchorEl}
+                transformOrigin={{
+                  horizontal: 'right',
+                  vertical: 'top',
+                }}
+              >
+                {typeof actionButtonPopoverContent === 'function' &&
+                  actionButtonPopoverContent(() =>
+                    setactionButtonPopoverAnchorEl(null)
+                  )}
+                {typeof actionButtonPopoverContent !== 'function' && (
+                  <Box minWidth={200}>
+                    <MenuList>
+                      {actionButtonPopoverContent.map((item, index) => (
+                        <MenuItem
+                          key={index}
+                          disabled={item.disabled}
+                          divider={item.divider}
+                          onClick={() => {
+                            item.onClick();
+                            setactionButtonPopoverAnchorEl(null);
+                          }}
+                        >
+                          {item.startIcon && (
+                            <ListItemIcon>{item.startIcon}</ListItemIcon>
+                          )}
+                          <ListItemText>{item.label}</ListItemText>
+                          {item.endContent && (
+                            <Typography color="secondary" variant="body2">
+                              {item.endContent}
+                            </Typography>
+                          )}
+                        </MenuItem>
+                      ))}
+                    </MenuList>
+                  </Box>
+                )}
+              </Popover>
+            )}
+            {showEllipsisMenu && (
+              <ZUIMenu
+                anchorEl={ellipsisMenuAnchorEl}
+                menuItems={ellipsisMenuItems}
+                onClose={() => setEllipsisMenuAnchorEl(null)}
+              />
+            )}
+          </>
+        )}
+      </Box>
+      {showBottomRow && (
+        <Box alignItems="center" display="flex" paddingTop={1}>
+          {(belowTitle || metaData) && (
+            <Box alignItems="center" display="flex">
+              {belowTitle && belowTitle}
+              {belowTitle && metaData && (
+                <Divider
+                  flexItem
+                  orientation="vertical"
+                  sx={{ marginX: '16px', marginY: '6px' }}
```

Comment:
In the design I have added the divider as a fixed 22px divider specifically sized with the category select component in mind. So with the current marginY when we fit in the category select, which is the only component we know right now that we want to fit in this slot, it will be a little bit too short. 

Although I do think that you were right in implementing a marginY instead of a fixed height since the content in the slot might change in the future. But in order to get the right proportion with the category select component in mind, set this to 4px marginY instead (0.25rem).